cardinal/events/tick_results.go

```diff
@@ -0,0 +1,44 @@
+package events
+
+import (
+	"encoding/json"
+	"github.com/rotisserie/eris"
+	"pkg.world.dev/world-engine/cardinal/receipt"
+)
+
+type TickResults struct {
+	Tick     uint64
+	Receipts []receipt.Receipt
+	Events   [][]byte
```

Comment:
Tick results is serialized into bytes, but events which is already bytes is serialized again. We should leave it as a concrete type here, (perhaps any or map[string]any)

Reply:
Done, left as `any`